app/views/menu/help-menu.tmpl.html

```diff
@@ -0,0 +1,21 @@
+<span role="help-button">
+  <i class="fa fa-question-circle"></i>&nbsp;Help
+</span>
+<span class="menu-item-toggle" ng-click="openHelpContextMenu($event)">
+  <i class="fa fa-caret-down"></i>
+</span>
+<ul role="context-menu" class="menu-item-context" ng-show="contextHelpMenuOpen">
+  <li role="context-menu-item" ng-click="openLink('http://raml.org/spec')">RAML Specification</li>
+  <li role="context-menu-item" ng-click="openLink('http://raml.org')">RAML Website</li>
```

Comment:
No need to add sugar JS to a link. If you've seen this elsewhere, bring it up and see if it makes sense to remove it from there too.